JoinColony/colonyCDapp
# feat: changed buttons copy in cancel modal

## Description

- Changed buttons copy in cancel modal
- After talking with Arren we changed `cancel payment` to `confirm cancelation`
<img width="567" alt="image" src="https://github.com/user-attachments/assets/81b9109f-a86a-4d61-8e92-6e65fb97341a">


## Testing

* Create a new Staged payment action.
* Use the "Permissions" decision method.
* Confirm the details so that you are on the Funding step.
* Now cancel the payment.
* The cancel payment modal will open up.

Resolves https://github.com/JoinColony/colonyCDapp/issues/2948
